.github/workflows/sonarcloud.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         steps:
             # We create the status here and then update it to success/failure in the `report` stage
             # This provides an easy link to this workflow_run from the PR before Sonarcloud is done.
-            - uses: guibranco/github-status-action-v2@5ef6e175c333bc629f3718b083c8a2ff6e0bbfbc
+            - uses: guibranco/github-status-action-v2@7ca807c2ba3401be532d29a876b93262108099fb
```

Comment:
Possibly a merge has gone wrong here and I think you might be unintentionally reverting something?

Reply:
this has been fixed on the main branch of the PR and is only present for the v0.8.0-rc.1 branch.